pymc_experimental/distributions/continuous.py

```diff
@@ -221,6 +221,163 @@ def moment(rv, size, mu, sigma, xi):
         return mode
 
 
+# Generalized Pareto Distribution
+class GenParetoRV(RandomVariable):
```

Comment:
Should subclass `ScipyRandomVariable` because Scipy RVs (sometimes) do something dumb with `size=(1,)`

```suggestion
class GenParetoRV(ScipyRandomVariable):
```